### ISSUE TITLE
configure with special comments

### ISSUE THREAD

**Issue by hmoffatt** [adrienverge/yamllint#409]
yamllint already allows checks to be disabled using `# yamllint ...` comments. Would it be possible to extend this to allow configuration, for example of the line-length max value?

**Comment by adrienverge** [comment]
Hello, you mean something like this?
```yaml
# yamllint config-line rule:line-length:{max: 30}
- Lorem ipsum dolor sit amet,
# yamllint config-line rule:line-length:{max: 80}
- consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore
```

It's not on the agenda. Handling configuration from various sources is already tricky, extending it to detect it from the YAML source would be even trickier and harder to maintain.

**Comment by hmoffatt** [comment]
Yes that was what I was thinking, although not necessarily on a line by line basis. Since you can already write ```# yamllint disable rule:line-length``` it seems logical to be able to configure the line length in the yaml file as well, thus keeping the configuration with the file itself.

Bundling a configuration file in the same directory will do though.
